7_react/react1/week2/todo-list-app/src/TodoHeader.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const TodoHeader = () => {
```

Comment:
Wow this is a great way of thinking in React! Separating components even if they are small 🎉